### PR TITLE
[conflict_resolver] loading data is incomplete

### PR DESCRIPTION
**Pull request by kongtiaowang**
#2878 has wrong rebase.
Redmine ticket is here.
https://redmine.cbrain.mcgill.ca/issues/12873

### CODE REVIEW

File: modules/conflict_resolver/php/NDB_Menu_Filter_Form_conflict_resolver.class.inc [hunk 1]
```diff
@@ -389,75 +389,7 @@ class NDB_Menu_Filter_Form_Conflict_Resolver extends NDB_Menu_Filter_Form
...
     }
-    /**
-     * Returns a list of candidates, users, etc.
-     * Need override for when Examiner field is conflicting,
-     * need Examiner's full name
-     *
-     * @return array
-     * @access private
-     */
-    function _getFullList()
```
Comment (davidblader) [inline modules/conflict_resolver/php/NDB_Menu_Filter_Form_conflict_resolver.class.inc]: what exactly was the original problem? this appears to be undoing a change i made that was necessary for CCNA

Comment (kongtiaowang) [inline modules/conflict_resolver/php/NDB_Menu_Filter_Form_conflict_resolver.class.inc]: It alway shows 25 pieces of record in the table.

File: modules/conflict_resolver/php/NDB_Menu_Filter_Form_conflict_resolver.class.inc [hunk 2]
```diff
@@ -389,75 +389,7 @@ class NDB_Menu_Filter_Form_Conflict_Resolver extends NDB_Menu_Filter_Form
...
-        // apply ORDER BY filters
-        $query .= " ORDER BY ";
-        if (!empty($this->filter['order'])) {
-            $query .= $this->filter['order']['field']
-                ." ".$this->filter['order']['fieldOrder'].", ";
-        }
-        $query .= $this->order_by;
-        // get the list
-
-        $pageNum = 1;
```
Comment (davidblader) [inline modules/conflict_resolver/php/NDB_Menu_Filter_Form_conflict_resolver.class.inc]: @kongtiaowang i discovered the issue
in this file, remove lines 422-425, 431-432, and alter line 433 to remove the `$limit` append

then do the same in the other file

**Comment by kongtiaowang** [comment]
@davidblader 
I will close this PR
because https://github.com/aces/Loris/pull/3000 do the same fix.

**Pull request closed without merging**